DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
 
-```java
-File precompiledPDF = new File("<path to your file>");
-LetterResponse response = client.sendPrecompiledLetter("Your reference", precompiledPDF)
+# Get message status
 
-```
-</details>
+The possible status of a message depends on the message type.
 
-#### Response
+## Status - text and email
 
-If the request is successful, the LetterResponse is returned from the client. Attributes of the LetterResponse are listed below.
+### Created
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+The message is queued to be sent to the provider. The notification usually remains in this state for a few seconds.
 
-```java
-	UUID notificationId;
-	<String> reference;
-```
+### Sending
 
-Otherwise the client will raise a `NotificationClientException`:
+The message is queued to be sent by the provider to the recipient, and GOV.UK Notify are waiting for delivery information.
 
-|`error.status_code`|`error.message`|
-|:---|:---|
-|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type live of 10 requests per 20 seconds"`<br>`}]`|
-|`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters with a team api key"`<br>`]}`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|
-|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "personalisation address_line_1 is a required property"`<br>`}]`|
+### Delivered
 
-</details>
+The message was successfully delivered.
 
-#### Arguments
+### Sent (text message only)
 
-<details>
-<summary>Click here to expand for more information.</summary>
+The text message was delivered internationally. GOV.UK Notify may not receive additional status updates depending on the recipient's country and telecoms provider.
 
-#### `reference`
-A unique identifier for the notification or an identifier for a batch of notifications. `reference` should not contain any personal information.
+### Failed
 
-#### `content`
-A base64 encoded string containing the file to be sent as a letter.
+This covers all failure statuses:
 
-</details>
+- `permanent-failure` - "The provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list"
+- `temporary-failure` - "The provider was unable to deliver message, email inbox was full or phone was turned off; you can try to send the message again"
+- `technical-failure` - "Notify had a technical failure; you can try to send the message again"
 
+## Status - letter
 
-## Get the status of one message
+### Failed
+
+The only failure status that applies to letters is `technical-failure` - Notify had an unexpected error while sending to our printing provider.
+
+### Accepted
+
+Notify is printing and posting the letter.
```

Comment:
We also have:
Received
The provider has received the letter to deliver.

Reply:
Changed

Reply:
Does this apply to Python as well?

Reply:
Yes, change python as well